supply/lib/supply/uploader.rb

```diff
@@ -83,20 +87,39 @@ def upload_screenshots(language)
       end
     end
 
+    def upload_binaries
```

Comment:
could this code be tighted up a little bit? maybe one `upload_binaries` method that expects an array and in the single apk case we just pass it `[Supply.config[:apk_path]]`?